tools/lint/main_ni.c

```diff
@@ -287,6 +287,9 @@ help(int shortout)
             "                create an exact YANG schema context. If specified, the '-F'\n"
             "                parameter (enabled features) is ignored.\n\n");
 
+    printf("  -X, --xpath-in-leafref\n"
+            "                Allow usage of XPath within leafref\n\n");
+
```

Comment:
As I said previously, these names you are using are really inaccurate. Leafref is already a subset of XPath. What this option does is enable `deref()` in leafref, nothing else.

Reply:
Done